ert_shared/storage/client.py

```diff
@@ -144,26 +144,35 @@ def data_for_key(self, case, key):
 
             response = ref_request(resp["ref_url"])
 
-            for real in response["realizations"]:
+            import time
```

Comment:
I am guessing this is for debugging purposes?

Reply:
Yes, will remove